pkg/inspector/image-inspector.go

```diff
@@ -135,7 +117,21 @@ func (i *defaultImageInspector) Inspect() error {
 		}
 	}
 
-	if i.imageServer != nil {
+	if len(i.opts.Serve) > 0 {
```

Comment:
This would indicate that there is no use case where you might call something on the image server without ever calling inspect first.  I *think* that is a valid use case now but are we certain it will hold true in the future?  Is it possible that we should not be manipulating the `dstPath` in the inspect call, using a pointer so changes are reflected, or be using another var if an integrator is expecting `dstPath` to be something specific?  

If the assumptions above will hold true then I have no issue with the move.